src/msg/async/dpdk/DPDK.cc

```diff
@@ -864,6 +871,24 @@ void DPDKQueuePair::process_packets(
   perf_logger->inc(l_dpdk_qp_rx_bytes, bytes);
 }
 
+void DPDKQueuePair::rx_start() {
+  last_inactive = clock_type::now();
+  rx_intr_enabled = false;
+  turn_off_rx_intr = [this] {
+    if (rx_intr_enabled) {
+      last_inactive = clock_type::now();
+      rte_eth_dev_rx_intr_disable(_dev_port_idx,_qid);
+      rx_intr_enabled = false;
+      ldout(cct,10) << __func__ << "turn off irq " << dendl;
```

Comment:
might want to add a space before `,` and before `turn off irq`.

also please remove the trailing space after `irq`.